packages/asset-uploader/src/db/load.ts

```diff
@@ -25,26 +24,27 @@ export const loadAssetsByProject = async (
     );
   }
 
-  const assets = await prisma.asset.findMany({
-    select: {
-      file: true,
-      id: true,
-      projectId: true,
-    },
-    where: {
-      projectId,
-      file: { status: "UPLOADED" },
-    },
-    orderBy: {
-      file: { createdAt: "desc" },
-    },
-  });
+  const assets = await context.postgrest.client
+    .from("Asset")
+    .select(
+      `
+        assetId:id,
+        projectId,
+        file:File!inner (*)
+      `
+    )
+    .eq("projectId", projectId)
+    .eq("file.status", "UPLOADED");
 
-  return assets.map((asset) =>
-    formatAsset({
-      assetId: asset.id,
-      projectId: asset.projectId,
-      file: asset.file,
-    })
-  );
+  const result: Asset[] = [];
+  for (const { assetId, projectId, file } of assets.data ?? []) {
+    if (file) {
+      result.push(formatAsset({ assetId, projectId, file }));
+    }
+  }
+
+  // order by createdAt desc
+  result.sort((left, right) => right.createdAt.localeCompare(left.createdAt));
```

Comment:
What the need of sorting? If stability then `id` must be used too

Reply:
Looks like we do this on client anyway, maybe not needed